app/src/main/java/app/waveflow/playback/CoilBitmapLoader.kt

```
@@ -0,0 +1,100 @@
+     * ça la requête continuerait pour une pochette dont personne ne veut plus.
+     */
+    private fun loadAsync(block: suspend () -> Bitmap): ListenableFuture<Bitmap> {
+        val future = SettableFuture.create<Bitmap>()
+
+        val job = scope.launch {
+            try {
+                future.set(block())
+            } catch (cancellation: CancellationException) {
+                future.cancel(/* mayInterruptIfRunning = */ false)
+                throw cancellation
+            } catch (error: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)